### DISCUSSION TITLE
Is there anyone worked on support milvus?

### DISCUSSION THREAD

**Discussion by maxiao124** [Knowly-ai/langtorch#139] [Ideas]
If not，need me to do it? @li2109

**Comment by li2109** [comment]
> If not，need me to do it? @li2109 

That would be fantastic!

**Reply by maxiao124** [reply]
happy to do it

**Reply by li2109** [reply]
Sent you a discord message.😃

**Reply by maxiao124** [reply]
Sorry for not following the project recently.

when I use retrofit，some stuck points have appeared，This comes from the interface design of Milvus，some Get request requires transmission of request body，such as:

1. Check if a collection exists（https://milvus.io/docs/check_collection.md）
2. Release a Collection（https://milvus.io/docs/release_collection.md）

In addition, there are also Delete Entities（https://milvus.io/docs/delete_data.md）need request body.
If run the test，you will get exception like : java.lang.IllegalArgumentException: method GET must not have a request body

This makes me think that perhaps using SDK directly would be more appropriate（https://github.com/milvus-io/milvus-sdk-java）
Is this a good  idea?  @li2109

**Reply by li2109** [reply]
sure
